src/datashare/azext_datashare/manual/_params.py

```diff
@@ -0,0 +1,332 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+# pylint: disable=unused-import
+
+from knack.arguments import CLIArgumentType
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type,
+    get_datetime_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group, validate_file_or_dict
+from azext_datashare.vendored_sdks.datashare.models._data_share_management_client_enums import ShareKind, Kind, SynchronizationMode, SynchronizationKind, RecurrenceInterval
+from azext_datashare.manual._validators import invitation_id_validator
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('datashare account list') as c:
+        c.argument('resource_group_name', resource_group_name_type)  # modified
+
+    with self.argument_context('datashare account show') as c:
+        c.argument('resource_group_name', resource_group_name_type)  # modified
```

Comment:
Why do we need to add `resource_group_name` ? The default definition is not enough ? And what does `#modified` in the last part means ?

Reply:
This is autogenerated. I think either keeping it or removing it is fine. #modified is to mark the code that was modified for autogenerated code since I copied all generated code here. The initial thought was to generate multiple times and we will have less manual code. We can only keep modified code in manual folder. But that's not so convenient for datashare with a large amount of code. I finally decided to remove the generated code that needs to be modified.